templates/workspaces/base/terraform/workspace.tf

```diff
@@ -31,6 +31,6 @@ module "aad" {
   depends_on = [
     azurerm_key_vault_access_policy.deployer,
     azurerm_key_vault_access_policy.resource_processor,
-    azurerm_private_endpoint.kvpe
+    null_resource.wait_for_dns_vault
```

Comment:
What is this for?

Reply:
When you write items to the keyvault, you have to do it over the private endpoint. However, there is a delay between the private endpoint being created and then that ip address being added to the DNS. This is just a nasty sleep for 2 minutes for the DNS to resolve.